### ISSUE TITLE
Historical DSWx-S1 Request over Alberta

### ISSUE THREAD

**Issue by cmarshak** [nasa/opera-sds#49]
### Venue

PST

### Product

DSWx-S1

### Input Data

https://github.com/OPERA-Cal-Val/DSWx-Validation-Experiments/blob/main/marshak/dswx-s1-organization/bc_alberta_request.csv

This has the input SLCs and the associated MGRS tiles (though duplicates are removed so only one mgrs tile for each slc is shown).

### Share Results

- [ ] Google Earth Engine (GEE)
- [ ] NASA Distributed Active Archive Centers (DAAC) User Acceptance Testing (UAT)

### Additional Notes

For Bruce Chapman

**Comment by niarenaw** [comment]
All products were generated except for 12VVK and 12VVL on 20170509. After an initial investigation, this appears to be due to an indexing issue in the SAS code. SDS is providing details to ADT.

**Comment by niarenaw** [comment]
Closing as missing date/tile pairs are not needed for analysis.

**Comment by niarenaw** [comment]
Will reprocess with final DSWx-S1 release for IV validation.

**Comment by philipjyoon** [comment]
We have to wait for Request #53 to be completed before we can start this.

**Comment by philipjyoon** [comment]
Reconfiguring the cluster to disable use of credentials in downloading data. This is necessary to download files from ASF UAT (as opposed to ASF OPS where we need to use credentials)

**Comment by philipjyoon** [comment]
Unfortunately the list of RTC products generated store in the GRQ ES was lost when the PST cluster was re-deployed. Fortunately I was able to find them directly on ASF UAT CMR using the following command
`python daac_data_subscriber.py survey -c OPERA_L2_RTC-S1_V1 --endpoint=UAT --start-date=2017-05-01T08:17:50Z --end-date=2018-10-01T08:35:59Z --step-hours=168 --use-temporal`

The files listed are here and this is what will be used to to trigger DSWx-S1 processing
[PST_request_49_list_of_RTC.csv](https://github.com/user-attachments/files/17120093/PST_request_49_list_of_RTC.csv)

**Comment by philipjyoon** [comment]
DSWx-S1 processing has started

**Comment by philipjyoon** [comment]
Processing is about 50% complete already. Should complete tonight

**Comment by philipjyoon** [comment]
This processing is complete. Here are the listing of the products:
[Req_49_dswx_s1_historical_Alberta.txt](https://github.com/user-attachments/files/17122985/Req_49_dswx_s1_historical_Alberta.txt)
